### ISSUE TITLE
[Question / Feature request] Shortcut that will open the auto-type window

### ISSUE THREAD

**Issue by renyhp** [keeweb/keeweb#1433]
I apologize in advance for opening a possibly useless issue.

After discussion in #1329 and #943, [there seemed to be a decision](https://github.com/keeweb/keeweb/issues/943#issuecomment-549115763) about this feature. Since most of the discussion happened in a closed issue, I'm opening this one just to ask for news about it, whether this feature was eventually discarded or this will be implemented.

I also apologize if this seems to rush developing, which is absolutely not my intention - I just want to ask a genuine question. Thank you very much

**Comment by antelle** [comment]
Hi! Now we have a setting for auto-type ("If only one matching entry is found, select that one automatically for Autotype") that allows to adjust the behavior:

-  on: the selection window will be shown if there's no match
- off: always show the selection window

In theory, there can be another shortcut that will disable matching when it's enabled, for example, the same but with shift or similar.

**Comment by renyhp** [comment]
> In theory, there can be another shortcut that will disable matching when it's enabled, for example, the same but with shift or similar.

Yes, that's exactly what I'm talking about here. The milestones and labels you added do answer my question, thank you!
